### ISSUE TITLE
Javascript appears twice in language selector

### ISSUE THREAD

**Issue by mohityadav7** [prvnbist/snippify#10]
Javascript syntax highlighting is used for .js files and .jsx files, so it appears twice in language selector dropdown.
It seems that fixing this issue may require changing the structure of languages object in state.js

**Comment by prvnbist** [comment]
Yes but I just changed the `jsx` key's value to `javascript-react` but monaco editor doesnt have syntax hightlighting support for `jsx` at the moment unless we build custom support for `jsx` syntax highlighting into monaco ourselves
